src/main/java/com/exasol/ReleaseRobot.java

```diff
@@ -75,6 +75,7 @@ private boolean hasBranch() {
         return this.gitBranch == null || this.gitBranch.isEmpty();
     }
 
+    // [impl->dsn~rr-starts-release-only-if-all-validation-succeed~1]
```

Comment:
This requirement is not visible in the code here. If you want to make that clear, let "validate" provide a return value representing the validation status and put the release in an if clause.

Reply:
I'm going to collect failed validation to a report and return it here, so it will be done, but not on this branch. P.S. I have a ticket for that

Reply:
Great. :-)
PS: if you say "I have a ticket", please put the issue ID in the comment. :-p